include/grpc/grpc_security.h

```diff
@@ -119,6 +119,18 @@ GRPCAPI void grpc_ssl_session_cache_destroy(grpc_ssl_session_cache* cache);
 GRPCAPI grpc_arg
 grpc_ssl_session_cache_create_channel_arg(grpc_ssl_session_cache* cache);
 
+/** --- grpc_call_credentials object.
```

Comment:
optional: This comment seems need to be a description for the defined struct. Maybe something like "gRPC Call Credentails"?

Reply:
This is line isn't new; it just moved up in the file. Though I agree with you in principle.